opts.go

```diff
@@ -39,14 +45,33 @@ func WithPluginDir(dirs []string) ConfigOptions {
 	}
 }
 
+// WithPluginConfDir can be used to configure the
+// cni configuration directory.
 func WithPluginConfDir(dir string) ConfigOptions {
 	return func(c *libcni) error {
 		c.pluginConfDir = dir
 		return nil
 	}
 }
 
-func WithLoNetwork() ConfigOptions {
+// WithMinNetworkCount can be used to configure the
+// minimum networks to be configured and initalized
+// for the status to report success. By default its 1.
+func WithMinNetworkCount(count int) ConfigOptions {
+	return func(c *libcni) error {
+		c.networkCount = count
+		return nil
+	}
+}
+
+// LoadOptions can be used with Load API
+// to load network configuration from different
+// sources.
+type LoadOptions func(c *libcni) error
```

Comment:
Nit, I'd expect this to be singular.
Also this is an exported type which receives an non-exported type, suprised linter doesn't catch this.

Reply:
Its not singular. I could do Load(WithLoNetwork,WithConf(abc),WithConflist(def))

Reply:
Yes, but `Load()` takes a list of options, each thing in the list is singular.

Reply:
I see what you mean there. Will update.

Reply:
done